### PR TITLE
Add Kerberos support for JDBC

### LINKED ISSUE #15: Tableau Kerberos Support
Add Kerberos Support to Tableau :

- Tableau Desktop , e.g. on Windows no user/password is needed
- Single Sign On for Tableau Server -> Kerberos Delegation

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #15

### CODE REVIEW

**Comment by kaklakariada** [comment]
# Test reports
## JDBC TDVT
```
$ python -m tdvt.tdvt run exasol_jdbc
Setting tdvt thread count to: 6
Testing exasol_jdbc
thread setting in exasol_jdbc.ini = 1
Starting smoke tests. Creating 1 worker threads.

Running exasol_jdbc StaplesConnectionTestexasol_jdbc 11

.
Finished exasol_jdbc StaplesConnectionTestexasol_jdbc 11

Running exasol_jdbc CastCalcsConnectionTestexasol_jdbc 11

.
Finished exasol_jdbc CastCalcsConnectionTestexasol_jdbc 11

2 smoke test(s) ran. 0 smoke tests disabled.
Smoke tests ran in 13.89 seconds.

Starting tests. Creating 1 worker threads.
Running exasol_jdbc logical.calcs.exasol_jdbc 1

...............................
Finished exasol_jdbc logical.calcs.exasol_jdbc 1

Running exasol_jdbc logical.staples.exasol_jdbc 2

..........................................
Finished exasol_jdbc logical.staples.exasol_jdbc 2

Running exasol_jdbc logical.lod.exasol_jdbc 3

...........................
Finished exasol_jdbc logical.lod.exasol_jdbc 3

Running exasol_jdbc logical.union.exasol_jdbc 4

.
Finished exasol_jdbc logical.union.exasol_jdbc 4

Running exasol_jdbc expression.standard.exasol_jdbc 5

...............................................................................................................................................................................................................
Finished exasol_jdbc expression.standard.exasol_jdbc 5

Running exasol_jdbc expression.lod.exasol_jdbc 6

.
Finished exasol_jdbc expression.lod.exasol_jdbc 6

Running exasol_jdbc expression.staples.exasol_jdbc 7

....
Finished exasol_jdbc expression.staples.exasol_jdbc 7

Running exasol_jdbc expression.median.exasol_jdbc 8

.
Finished exasol_jdbc expression.median.exasol_jdbc 8

Running exasol_jdbc expression.percentile.exasol_jdbc 9

.
Finished exasol_jdbc expression.percentile.exasol_jdbc 9

Running exasol_jdbc calc_data.exasol_jdbc 10

.
Finished exasol_jdbc calc_data.exasol_jdbc 10

Running exasol_jdbc TestBadPassword.exasol_jdbc 11

.
Finished exasol_jdbc TestBadPassword.exasol_jdbc 11


Test Count: 921 tests
        Passed tests: 921
        Failed tests: 0
        Tests run: 921
        Disabled tests: 0
        Skipped tests: 0

Other information:
        Smoke test time: 13.89 seconds
        Main test time: 118.96 seconds
        Total time: 132.85 seconds
```

## ODBC TDVT

```
$ python -m tdvt.tdvt run exasol_odbc
Setting tdvt thread count to: 6
Testing exasol_odbc
thread setting in exasol_odbc.ini = 1
Starting smoke tests. Creating 1 worker threads.

Running exasol_odbc StaplesConnectionTestexasol_odbc 12

.
Finished exasol_odbc StaplesConnectionTestexasol_odbc 12

Running exasol_odbc CastCalcsConnectionTestexasol_odbc 12

.
Finished exasol_odbc CastCalcsConnectionTestexasol_odbc 12

2 smoke test(s) ran. 0 smoke tests disabled.
Smoke tests ran in 6.74 seconds.

Starting tests. Creating 1 worker threads.
Running exasol_odbc logical.calcs.exasol_odbc 1

...............................
Finished exasol_odbc logical.calcs.exasol_odbc 1

Running exasol_odbc logical.staples.exasol_odbc 2

..........................................
Finished exasol_odbc logical.staples.exasol_odbc 2

Running exasol_odbc logical.lod.exasol_odbc 3

...........................
Finished exasol_odbc logical.lod.exasol_odbc 3

Running exasol_odbc logical.union.exasol_odbc 4

.
Finished exasol_odbc logical.union.exasol_odbc 4

Running exasol_odbc expression.standard.exasol_odbc 5

...............................................................................................................................................................................................................
Finished exasol_odbc expression.standard.exasol_odbc 5

Running exasol_odbc expression.lod.exasol_odbc 6

.
Finished exasol_odbc expression.lod.exasol_odbc 6

Running exasol_odbc expression.staples.exasol_odbc 7

....
Finished exasol_odbc expression.staples.exasol_odbc 7

Running exasol_odbc expression.median.exasol_odbc 8

.
Finished exasol_odbc expression.median.exasol_odbc 8

Running exasol_odbc expression.percentile.exasol_odbc 9

.
Finished exasol_odbc expression.percentile.exasol_odbc 9

Running exasol_odbc calc_data.exasol_odbc 10

.
Finished exasol_odbc calc_data.exasol_odbc 10

Running exasol_odbc TestBadPassword.exasol_odbc 11

.
Finished exasol_odbc TestBadPassword.exasol_odbc 11


Test Count: 921 tests
        Passed tests: 921
        Failed tests: 0
        Tests run: 921
        Disabled tests: 0
        Skipped tests: 0

Other information:
        Smoke test time: 6.74 seconds
        Main test time: 84.3 seconds
        Total time: 91.04 seconds
```

**Review by AnastasiiaSergienko — COMMENTED** [review]
A few small findings :)

File: doc/user_guide/user_guide.md [hunk 1]
```diff
@@ -1,68 +1,133 @@
...
   * Linux: `/opt/tableau/tableau_driver/jdbc`
 
-**Important note for Windows:** Make sure to download and install file `EXASOL_JDBC-<version>.msi`. The JDBC driver will be installed to `C:\Program Files\Exasol\EXASolution-7.1\JDBC\exajdbc.jar`. Only the JDBC driver for Windows supports Kerberos under Windows.
+#### Install the ODBC Driver
 
-### With Tableau Desktop
+The connector for ODBC requres the Exasol ODBC driver to be installed for Tableau Desktop and Server.
 
-* Clone the Tableau-Connector repository. You can execute the following command from [Git Bash](https://gitforwindows.org/) terminal:
+Download and install the latest JDBC driver for your operating system from the [download page](https://www.exasol.com/portal/display/DOWNLOAD/).
```
Comment (AnastasiiaSergienko) [inline doc/user_guide/user_guide.md:36]:
```suggestion
Download and install the latest ODBC driver for your operating system from the [Exasol download page](https://www.exasol.com/portal/display/DOWNLOAD/).
```

File: doc/user_guide/user_guide.md [hunk 2]
```diff
@@ -1,68 +1,133 @@
...
-    Example:
+1. Copy the connector `.taco` files to
+  * `C:\Users\[Windows User]\Documents\My Tableau Repository\Connectors` (Windows)
+  * `/Users/[user]/Documents/My Tableau Repository/Connectors` (macOS)
+2. As the connectors are not signed, you need to start Tableau Desktop with command line argument `-DDisableVerifyConnectorPluginSignature`, e.g. by creating a `.bat` file with the following content:
 
-    ```shell
-    "C:\Program Files\Tableau\Tableau <version>\bin\tableau.exe" -DConnectPluginsPath=C:\Users\user\git\tableau-connector\src
+    ```bat
+    "C:\Program Files\Tableau\Tableau 2021.3\bin\tableau.exe" -DDisableVerifyConnectorPluginSignature
```
Comment (AnastasiiaSergienko) [inline doc/user_guide/user_guide.md:56]: Maybe the version placeholder is better? You can forget to change this when the new Tableau version is released

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]
Two more tiny findings

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**